_targets.R

```diff
@@ -15,76 +15,90 @@ parameterCd <- '00010'
 startDate <- "2014-05-01"
 endDate <- "2015-05-01"
 fetch_out_folder <- "1_fetch/out"
+process_out_folder <- '2_process/out'
 
 ## Fetch and save downloaded nwis data info
 p1_targets_list <- list(
   tar_target(
-    site_data_01427207,
+    site_data_01427207_csv,
     download_nwis_data(site_num = site_1,
                        parameterCd = parameterCd,
                        startDate = startDate,
-                       endDate = endDate),
+                       endDate = endDate,
+                       fileout = file.path(fetch_out_folder,paste0('site_',site_1,".csv"))
+                       ),
+    
+    ## Changed to file format
+    format = "file"
   ),
   
   tar_target(
-    site_data_01432160,
+    site_data_01432160_csv,
     download_nwis_data(site_num = site_2,
                        parameterCd = parameterCd,
                        startDate = startDate,
-                       endDate = endDate),
+                       endDate = endDate,
+                       fileout = file.path(fetch_out_folder,paste0('site_',site_2,".csv"))),
+    ## Changed to file format
+    format = "file"
   ),
 
   tar_target(
-    site_data_01436690,
+    site_data_01436690_csv,
     download_nwis_data(site_num = site_3,
                        parameterCd = parameterCd,
                        startDate = startDate,
-                       endDate = endDate),
+                       endDate = endDate,
+                       fileout = file.path(fetch_out_folder,paste0('site_',site_3,".csv"))),
+    ## Changed to file format
+    format = "file"
   ),
 
   tar_target(
-    site_data_01466500,
+    site_data_01466500_csv,
     download_nwis_data(site_num =  site_4,
                        parameterCd = parameterCd,
                        startDate = startDate,
-                       endDate = endDate),
+                       endDate = endDate,
+                       fileout = file.path(fetch_out_folder,paste0('site_',site_4,".csv"))),
+    ## Changed to file format
+    format = "file"
   ),
 
   tar_target(
     site_data,
-    combine_nwis_data(site_data_01427207,
-                      site_data_01432160,
-                      site_data_01436690,
-                      site_data_01466500),
+    combine_nwis_data(fetch_out_folder)
   ),
   
+  ## changed to object format
   tar_target(
     site_info_csv,
     nwis_site_info(fileout = file.path(fetch_out_folder, 'site_info.csv'),
```

Comment:
Now that the target is an object target, our convention is that its name should no longer end in `_csv`.

Additionally, `nwis_site_info()` no longer needs to take a file path argument, since it no longer needs to write a CSV. Can you please rename this target, and change `nwis_site_info()` so that it no longer takes a CSV argument or writes a CSV?